.gitignore

```diff
@@ -39,6 +39,10 @@ tmp/
 logs/
 *.log
 
+# generated
+strongbox-web-core/strongbox-vault
+strongbox-web-core/strongbox
```

Comment:
Could you please revert these two lines? Thanks for pointing this out. We'll fix it. It shouldn't be ignored through here.

Reply:
ok done